Assets/HoloToolkit/Utilities/Scripts/ManualCameraControl.cs

```diff
@@ -218,13 +222,13 @@ private void ManualCameraControl_MouseLookTick(ref Vector3 rot)
 
         if (UnityEngine.Cursor.lockState == CursorLockMode.Locked)
         {
-            Debug.Log("Cursor locked state");
+            //Debug.Log("Cursor locked state");
```

Comment:
I agree. We need to be careful with putting too many log statements in. They should really be reserved for error messages to avoid cluttering up the debug window. (You could define your own conditional compile statement for testing purposes that is not part of the standard build).